.drone.yml

```diff
@@ -358,6 +359,8 @@ steps:
     when:
       target: PROD
       event: promote
+    depends_on:
+      - sanity_check_build_prod
 
   # Deploy to Production environment
   - name: deploy_to_prod
```

Comment:
if i remember correctly this will need to depend on `clone_repos_prod` otherwise if it starts before clone_repos_prod completes, it fails

Reply:
clone_repos_prod dependency hasn't been added